packages/order-utils/src/order_validation_utils.ts

```diff
@@ -198,10 +208,9 @@ export class OrderValidationUtils {
             throw new Error(OrderError.InvalidSignature);
         }
         const filledTakerTokenAmount = await this._orderFilledCancelledFetcher.getFilledTakerAmountAsync(orderHash);
-        OrderValidationUtils._validateRemainingFillAmountNotZeroOrThrow(
-            signedOrder.takerAssetAmount,
-            filledTakerTokenAmount,
-        );
+        if (signedOrder.takerAssetAmount.eq(filledTakerTokenAmount)) {
+            throw new Error(RevertReason.OrderUnfillable);
```

Comment:
Let's not throw `OrderUnfillable`. It's cryptic. How about `REMAINING_FILL_AMOUNT_ZERO`?

Reply:
This method is _only_ used by our combinatorial tests and as such they must mimick the behavior of the actual contract. The validation method exposed in `@0x/contract-wrappers` does not throw this cryptic error.